app/wasm/testdata/version.txt

```diff
@@ -1 +1 @@
-v0.5.0
```

Comment:
asking to learn - how is the tesdata generated? Is there a repo/code I can look at?

Reply:
https://github.com/osmosis-labs/osmosis/blob/main/app/wasm/testdata/download_releases.sh

Reply:
As part of this, can create a new release of osmosis-bindings, and update version.txt here. Will do.